pages/docs/data-ops/resource-identifiers.mdx

```diff
@@ -72,17 +71,6 @@ the Hashboard web app to be code-controlled, without recreating it.
 Anywhere you specify a filepath in a Hashboard config, you may instead specify a GRN.
 This allows you to reference resources independently of how you structure your files.
 
-### Referencing dbt models
-
-When referencing models imported by the [dbt integration](./dbt-integration.mdx), the standard GRN format is valid.
-It is also possible to reference dbt models using the unique identifier of the dbt node representing the model:
-
-```bash
-dbt::dbtuniqueid
-```
```

Comment:
wait this seems cool. Does this feature not work anymore? Did it ever?

Reply:
seems cool but doesnt work anymore idk if it ever did tbh cuz I dont remember seeing any code that would make this happen lol